src/Http/ServerRequest.php

```diff
@@ -354,6 +354,9 @@ protected function _setConfig($config)
         $this->url = substr($uri->getPath(), 1);
         $this->here = $this->base . '/' . $this->url;
 
+        $this->query = $this->_processGet($config['query'], $querystr);
```

Comment:
Do we still need to pass `querystr` here? If the querystr has been set into the uri object, we'll do some duplicate work now.

Reply:
At this point `$this->uri` object won't have the required query string set if the URL was specified using `url` config in constructor.